doc/develop/test/twister.rst

```diff
@@ -972,6 +973,43 @@ robot_testsuite: <robot file path> (default empty)
 robot_option: <robot option> (default empty)
     One or more options to be send to robotframework.
 
+Power
+=====
+The ``power`` harness is used to measure and validate the current consumption.
+It integrates with 'pytest' to perform automated data collection and analysis using a hardware power monitor.
+
+The harness executes the following steps:
+1. Initializes a power monitoring device (e.g., ``stm_powershield``) via the ``PowerMonitor`` abstract interface.
+2. Starts current measurement for a defined ``measurement_duration``.
+3. Collects raw current waveform data.
+4. Uses a peak detection algorithm to segment data into defined execution phases based on power transitions.
+5. Computes RMS current values for each phase using a utility function.
+6. Compares the computed values with user-defined expected RMS values.
```

Comment:
```suggestion
The harness executes the following steps:

1. Initializes a power monitoring device (e.g., ``stm_powershield``) via the ``PowerMonitor`` abstract interface.
#. Starts current measurement for a defined ``measurement_duration``.
#. Collects raw current waveform data.
#. Uses a peak detection algorithm to segment data into defined execution phases based on power transitions.
#. Computes RMS current values for each phase using a utility function.
#. Compares the computed values with user-defined expected RMS values.
```

Reply:
Can you actually apply the full suggestion please? This makes the list easier to re-arrange (additions, deletions, etc) without causing unnecessary diffs.
Thanks!